src/tests/fs_tests.rs

```diff
@@ -2552,17 +2552,14 @@ pub mod fs_tests {
         // Create a directory
         assert_eq!(cage.mkdir_syscall("/getdents", S_IRWXA), 0);
 
-        // Open the directory
-        let fd = cage.open_syscall("/getdents", O_RDWR, S_IRWXA);
```

Comment:
So, you removed this because the opened fd was never used before in this test?

Reply:
Yes, 
An invalid file descriptor value (-1) was passed to the getdents function, rendering this action redundant